autoload/go/config.vim

```diff
@@ -240,16 +240,40 @@ function! go#config#SetTemplateAutocreate(value) abort
   let g:go_template_autocreate = a:value
 endfunction
 
+function! go#config#Metalinter() abort
+  return get(g:, "go_metalinter", "gometalinter")
+endfunction
+
+function! go#config#MetalinterConfigEnabled() abort
+  return get(g:, "go_metalinter_config_enabled", 0)
+endfunction
+
+function! go#config#MetalinterGrepNew() abort
+  return get(g:, "go_metalinter_grep_new", 0)
+endfunction
```

Comment:
Neither `go#config#MetalinterConfigEnabled` nor `go#config#MetalinterGrepNew` is needed. vim-go doesn't need to support every possible option. If people want an escape hatch, there's already one in the form of `g:go_metalinter_command`.

Reply:
Sorry, agreed, I didn't intend to push that. I'm playing around with the best way to interact with the golangci-lint config. Config values are trumping flags which is a bit different than I expected.

Reply:
let's not worry about it; there's an escape hatch, and people can use the config as they see fit.